docs-src/cloud-context/saml-finish.md

```diff
@@ -0,0 +1,19 @@
+---
+id: saml-finish
+title: How to finish your SAML configuration
+sidebar_label: Finish SAML configuration
+description: To finish your SAML configuration, send the sign-in URL and certificate to us and then test your connection.
+tags:
+  - guide-context
+---
+
+After you configure SAML with your IdP, we can finish the configuration on our side.
+To do that, send the following information to us:
+
+- The sign-in URL from your application
+- The SAML sign-in certificate
+
+When you receive confirmation from us that we have finished configuration, log in to Temporal Cloud.
+This time, though, enter your email address in **Enterprise identity** and select **Continue**.
+Do not select **Continue with Google** or **Continue with Microsoft**.
+You will be redirected to the authentication page of your IdP.
```

Comment:
So--since the doc is more long-lived idk if we should add this but, the customer needs to let us know if they want to convert their account to be SAML, which means
1. their user invites will be SAML only
2. they can only login via SAML (google sso and microsoft login will be rejected)

However I can see this functionality changing in the future so maybe it's okay to leave it out of the docs

Reply:
Let's stet for now. This would be good to add later if/when needed.